gateway/gateway/schemas.py

```diff
@@ -13,23 +13,23 @@ class CreateOrderSchema(Schema):
     )
 
 
-class GetOrderSchema(Schema):
+class ProductSchema(Schema):
+    id = fields.Str()
+    title = fields.Str()
+    maximum_speed = fields.Int()
+    in_stock = fields.Int()
+    passenger_capacity = fields.Int()
```

Comment:
Is it worth separating the schemas into `CreateProductSchema` and `GetProductSchema`?
(maybe not in this case as they are identical... but should we have a think about how schemas are named?)